cloud/amazon/route53.py

```diff
@@ -344,6 +367,11 @@ def main():
           elif not alias_hosted_zone_id_in:
               module.fail_json(msg = "parameter 'alias_hosted_zone_id' required for alias create/delete")
 
+    if vpc_id_in and not private_zone_in:
```

Comment:
Use the "required_together" option if applicable --  see https://github.com/ansible/ansible-modules-core/blob/047d7f0775ea9c1fca0f6beecd468fc2e8280219/cloud/vmware/vsphere_guest.py#L1239-L1250